helpers/signatureVerification.ts

```diff
@@ -0,0 +1,121 @@
+/* 

```

Comment:
Please apply all comments on the functions in this file to the version in `misc.ts` and then delete this file. We don't need the functions in 2 places